src/sentry/incidents/endpoints/organization_incident_index.py

```diff
@@ -97,11 +97,9 @@ def get(self, request: Request, organization) -> Response:
             except InvalidParams as err:
                 return Response(str(err), status=status.HTTP_400_BAD_REQUEST)
 
-            team_filter_query = Q(
-                alert_rule__owner_id__in=teams_query.values_list("actor_id", flat=True)
-            )
+            team_filter_query = Q(alert_rule__team_id__in=teams_query.values_list("id", flat=True))
             if unassigned:
-                team_filter_query = team_filter_query | Q(alert_rule__owner_id=None)
+                team_filter_query = team_filter_query | Q(alert_rule__team_id=None)
```

Comment:
I wonder if this worked before... we'd usually use `alert_rule__owner_id__isnull=True`

Reply:
I'll switch to `isnull`.